lib/rule/RexGetValueRule.php

```diff
@@ -64,6 +65,11 @@ public function processNode(Node $methodCall, Scope $scope): array
         }
 
         $nameType = $scope->getType($args[0]->value);
+        $names = TypeUtils::getConstantStrings($nameType);
+        if (0 === count($names)) {
+            return [];
+        }
+
         $valueReflection = new RexGetValueReflection();
         if (null !== $valueReflection->getValueType($nameType, $methodReflection->getDeclaringClass()->getName())) {
```

Comment:
before this fix `$valueReflection->getValueType` returned `null` for a `non-falsy-string $nameType` and therefore triggered a error